.github/pull-request-template.md

```diff
@@ -38,6 +38,7 @@ Fixes:
 
 ## **Pre-merge author checklist**
 
+- [ ] I've read [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs/blob/main/docs/pull-requests.md)
```

Comment:
Looks like this change was included accidentally

Reply:
Oh sorry for that, I don't know what I did